rofi-bluetooth

```diff
@@ -305,7 +305,7 @@ show_menu() {
 }
 
 # Rofi command to pipe into, can add any options here
-rofi_command="rofi -dmenu -no-fixed-num-lines -yoffset -100 -i -p"
+rofi_command="rofi -dmenu $@"
```

Comment:
It might be a good idea to keep the -p prompt argument within this script.

Reply:
Performed The changes.
Check if this works.